### ISSUE TITLE
add dataset error

### ISSUE THREAD

**Issue by Data-Designer** [thuml/Transfer-Learning-Library#132]
Hello, I customized a data set, which fully complies with the specification, and modified the init file in common/vision/dataset and added the clef.py file. Why can't I recognize the data I added when running the code in the **example**.

_dann.py: error: argument -d/--data: invalid choice: 'Clef' (choose from 'Aircraft', 'Any', 'COCO70', 'CUB200', 'DTD', 'DomainNet', 'EuroSAT', 'ImageList', 'ImageNetR', 'ImageNetSketch', 'MNIST', 'MNISTRGB', 'Office31', 'OfficeCaltech', 'OfficeHome', 'Optional', 'OxfordFlowers102', 'OxfordIIITPet', 'PACS', 'PatchCamelyon', 'Resisc45', 'Retinopathy', 'SVHN', 'SVHNRGB', 'StanfordCars', 'StanfordDogs', 'Tuple', 'USPS', 'USPSRGB', 'VisDA2017', 'check_exits', 'download_data', 'amazon', 'camelyon17', 'civilcomments', 'iwildcam', 'ogb-molpcba', 'poverty', 'fmow', 'py150', 'rxrx1', 'globalwheat', 'celebA', 'domainnet', 'waterbirds', 'yelp', 'bdd100k', 'sqf', 'encode', 'Digits')_

**Comment by thucbx99** [comment]
Did you add this dataset in the `__all__` attribute of `init.py`?

**Comment by Atishaysjain** [comment]
Hello, 

I am facing the same issue as @Data-Designer. I have added the name of my dataset in the __all__ attribute of __init__.py. I have added the py file corresponding to my dataset in the tllib/vision/datasets/ folder and have imported the same in the init file.

Yet, I am facing the error: 
dann.py: error: argument -d/--data: invalid choice: 'Allaple_A_L' (choose from 'Aircraft', 'Any', 'CIFAR10', 'CIFAR100', 'COCO70', 'CUB200', 'Caltech101', 'DTD', 'DomainNet', 'EuroSAT', 'Food101', 'ImageList', 'ImageNetR', 'ImageNetSketch', 'MNIST', 'MNISTRGB', 'Office31', 'OfficeCaltech', 'OfficeHome', 'Optional', 'OxfordFlowers102', 'OxfordIIITPets', 'PACS', 'PatchCamelyon', 'Resisc45', 'Retinopathy', 'SUN397', 'SVHN', 'SVHNRGB', 'StanfordCars', 'StanfordDogs', 'Tuple', 'USPS', 'USPSRGB', 'VisDA2017', 'check_exits', 'download_data', 'Digits')

Can someone guide me on the same. Thanks.
